### ISSUE TITLE
wx.Slider on Linux Mint doesn't update indicator at times

### ISSUE THREAD

**Issue by Correct-Syntax** [wxWidgets/Phoenix#1855]
**Operating system**: Linux Mint 20
**wxPython version & source**:     4.1.0 from wxpython extras installed via pip     
**Python version & source**:      3.8, already installed on the system
      

**Description of the problem**:

```python
            self.slider = wx.Slider(
                parent,
                id=wx.ID_ANY,
                value=1,
                minValue=1,
                maxValue=100,
                style=wx.SL_HORIZONTAL | wx.SL_AUTOTICKS | wx.SL_LABELS
                )
            self.slider.SetTickFreq(10)
            sizer.Add(self.slider, flag=wx.EXPAND|wx.ALL, border=5)
            self.slider.Bind(
                wx.EVT_SCROLL_THUMBRELEASE,
                self.WidgetEvent
                )
```
In the screenshot below, you can see that the indicator doesn't the same amount as the circle button thingy:

![Screenshot from 2020-11-20 03-58-55](https://user-images.githubusercontent.com/60711001/100283709-984db880-2f33-11eb-990f-e6d08b90db9f.png)

It happens when adjusting the slider back and forth. When the problem happens, the circle thing (for lack of a better term) stays in the same place, but the indicator updates with the values (ex. 100). The strange thing is that this only happens sometimes.

**Comment by swt2c** [comment]
Can you reproduce this in the slider demo?  `demo/Slider.py`

**Comment by Correct-Syntax** [comment]
@swt2c, I was not able to reproduce it in the slider demo. It could have something to do with  #1854 as this widget is placed inside of that panel (see the code for that issue).

**Comment by swt2c** [comment]
Can you please make a complete, runnable example that reproduces the issue?

**Comment by Correct-Syntax** [comment]
@swt2c, Thank you for trying to help. I really appreciate it. :)

I am not sure if it counts, but maybe you could take a look at my project, https://github.com/Correct-Syntax/Gimel-Studio.  The issue happens with any of the nodes with that slider widget (contrast, etc). 

If you would prefer to just take a glance at the code in context, the relevant files are:

https://github.com/Correct-Syntax/Gimel-Studio/blob/master/src/GimelStudio/node/property.py (contains actual slider widget code)

https://github.com/Correct-Syntax/Gimel-Studio/blob/master/src/GimelStudio/interface/node_property_panel.py (contains panel where (slider) widgets are placed)

Maybe then you could tell me if It has to do with the way I am doing something, an issue with Linux Mint or if it really is an issue with wxpython. 

Thank you. 👍

**Comment by DietmarSchwertberger** [comment]
I seem to remember an earlier post of you. Do you create custom events or CallAfter at a high rate or do you have a heavy thread besides the main GUI thread?

Anyway I doubt that someone will work on this if there is not a minimum code sample to reproduce the problem.

**Comment by swt2c** [comment]
> I seem to remember an earlier post of you. Do you create custom events or CallAfter at a high rate or do you have a heavy thread besides the main GUI thread?

I was thinking the same way, Dietmar.  Possibly something that's preventing the GUI thread from drawing.

**Comment by Correct-Syntax** [comment]
@DietmarSchwertberger 

> Do you create custom events or CallAfter at a high rate or do you have a heavy thread besides the main GUI thread?

No. I couldn't get the threading to work from before, so no it's just the regular gui thread. :)

>  I doubt that someone will work on this if there is not a minimum code sample to reproduce the problem.

I can understand that. I could work on creating one, but that would involve a lot of code still. Could it be because of #1854? I am not sure if I am not clearing the panel correctly -Maybe then ids get confused or something?

@swt2c

> Possibly something that's preventing the GUI thread from drawing.

That is what I was thinking too.
